pkg/workspace_repo/generate_mocks.go

```diff
@@ -1,5 +1,5 @@
 package workspace_repo
 
 // Add mocks as necessary
-//go:generate mockgen -destination=mock_workspace.go -package=workspace_repo github.com/chanzuckerberg/happy/pkg/workspace_repo Workspace
-//go:generate mockgen -destination=mock_workspace_repo.go -package=workspace_repo github.com/chanzuckerberg/happy/pkg/workspace_repo WorkspaceRepoIface
+//go:generate mockgen -destination=mock_workspace_test.go -package=workspace_repo github.com/chanzuckerberg/happy/pkg/workspace_repo Workspace
```

Comment:
By naming a file `xxx_test.go`, it's excluded from code coverage; these mocks cannot be placed under the `mocks` package as this causes a circular dependency.